README.md

```diff
@@ -1,2 +1,123 @@
+
+### Components
+
+Most of the components in the starter theme are built from base components provided by [`@enact/ui`](https://github.com/enactjs/enact/tree/master/packages/ui).  The base components have minimal styling, just enough to ensure that the behavior and structure works, the rest is intentionally kept unassigned for the theme to define.
+
+* `BodyText` - simple component for displaying text
+* `Button` - a button; can support icons
+* `Checkbox` - a checkbox
+* `CheckboxItem` - a combined `ToggleItem` and `Checkbox`
+* `Heading` - a heading with optional divider line
+* `Icon` - an icon that can be used alone; also used by `Button` and other components
+* `Item` - a themed, generic item; great for lists of things
+* `Panels` - a component to handle multiple views; also exports `Panel`
+* `RadioItem` - a selectable `ToggleItem` with a radio button
+* `Skinnable` - a HOC for providing skin support to components
+* `SlotItem` - an `Item` that has "slots" where additional components (like an `Icon`) can be rendered
+* `ThemeDecorator` - a HOC for applying the theme's behaviors to an application
+* `ToggleIcon` - an `Icon` with `ui/Toggleable` behavior
+* `ToggleItem` - an `Item` with `ui/Toggleable` behavior
+
+### Behaviors
+
+The Enact framework has several modules that provide various behaviors.  The starter theme applies the following behaviors to the wrapped app by default:
+
+* Resolution independence - makes sure components are the same size (relative or not) on different screen resolutions
+* Spotlight - spatial navigation with pointer or other input devices
+
+### Styles
+
+The Enact framework uses [LESS](http://lesscss.org/) to create the styles that are applied to components.  Theme-wide variables/rules that should apply to all skins in the theme are kept in files in the `styles` directory.  Developers can mostly stick to `variables.less`, but additional functions can be added to `mixins.less`.
+
+#### Skins
+
+The way we've defined a "theme" vs. a "skin" is that a theme includes all of the components, behaviors, and collections of skins, while a skin is purely the styling, the appearance of those components.  Think of a theme as the model of a car and the skin as its paint job.
+
+The starter theme comes with one skin (`my-skin`) and the variables and colors for it are found in `variables-my-skin.less` and `colors-my-skin.less`, respectively.
+
+## Customizing
+
+### Copy
+
+The easiest way to get a fresh copy is to clone the [`my-theme` repo](https://github.com/enactjs/my-theme.git).  If an existing project already has `@enact/my-theme` it can be copied from there, as well.
+
+It is recommended to choose a new name for the theme and use it as the project directory for the copy.
+
+```
+git clone https://github.com/enactjs/my-theme.git uranium
+```
+or
+```
+mkdir uranium && cp -r path/to/node_modules/@enact/my-theme/* uranium 
```

Comment:
If the primary use case for this repo is, as above, to clone and modify, I think this is more confusing than useful. Would `@enact/my-theme` exist in `node_modules` and why? Even if that is the case, is this an option we want to promote (which is the effect by including it here)?